README.md

```
@@ -1,166 +1,301 @@
+pdm run test       # Run tests to ensure nothing broke
 ```
 
-## Publish
+Run these commands before submitting PRs to ensure code quality standards.
+
+## 📦 Publishing
 
-To publish new package (author only)
+For package maintainers:
 
+## 📦 Publishing
```

markdownlint-cli2 (0.17.2)

Multiple headings with the same content

(MD024, no-duplicate-heading)